File: official/docs/csharp/v5/referral-customers/refund-by-amount.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            EasyPost.Parameters.ReferralCustomer.RefundByAmount parameters = new()
            {
                Amount = 2000,
            };

            EasyPost.Models.API.PaymentRefund refund = await client.Beta.ReferralCustomer.RefundByAmount(parameters);

            Console.WriteLine(JsonConvert.SerializeObject(refund, Formatting.Indented));
        }
    }
}

File: official/docs/golang/current/referral-customers/refund-by-amount.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func refundByAmount() {
	client := easypost.New("EASYPOST_API_KEY")

	refund, _ := client.BetaRefundByAmount(2000)

	fmt.Println(refund)
}
